src/Controller/ElFinderController.php

```diff
@@ -226,7 +226,7 @@ public function mainJS()
     {
         $version = new EmptyVersionStrategy();
         $package = new Package($version);
-        $mainUrl = $package->getUrl('bundles/fmelfinder/js');
+        $mainUrl = $package->getUrl('/bundles/fmelfinder/js');
```

Comment:
I know this is already merged but shouldn't this code also contain the `assets_path`? see example config https://github.com/helios-ag/FMElfinderBundle#add-configuration-options-to-your-configyaml where we can define a asset path?

Reply:
I just confirmed for me when having `assets_path` set and chaning this line to:
```
        $mainUrl = $package->getUrl(sprintf('%s/bundles/fmelfinder/js', $this->params['assets_path']));
```
I loads my assets file for elfinder again